app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -132,14 +133,20 @@ public static CharSequence bold(CharSequence... content) {
     }
 
     public static void hideKeyboard(Activity activity) {
-        View view = activity.getCurrentFocus();
+        if (activity == null)
+            return;
 
```

Comment:
I guess just adding these two lines to the top of the method will solve the issue. The other additions are logically identical to the previous version.

Reply:
exactly, do you want me to undo the other changes or leave them (as they improve readability)?